src/http/cookie.cr

```diff
@@ -251,8 +251,14 @@ module HTTP
         expires = parse_time(match["expires"]?)
         max_age = match["max_age"]?.try(&.to_i64.seconds)
 
+        # Unwrap quoted cookie value
+        cookie_value = match["value"]
+        if cookie_value.starts_with?('"')
```

Comment:
To be extra sure:

```suggestion
        if cookie_value.starts_with?('"') && cookie_value.ends_with?('"')
```

Reply:
This code is copied from like 20 lines above (`def parse_cookies(header, &)`). In this case, both need to be changed.